tcf_website/views/search.py

```diff
@@ -110,3 +108,53 @@ def format_instructors(results):
         formatted.append(instructor)
 
     return formatted
+
+
+def build_elasticsearch_query(query, api_endpoint):
```

Comment:
I think we should abstract away the api_endpoints. 

Maybe add `search_courses` and `search_instructors` that wrap `build_elasticsearch_query` and use os.environ directly so we don't have to keep passing around `api_endpoint`? Then lines 21 and 22 would be `search_courses(query)` and `search_instructors(query)`

Reply:
done